pkgs/development/libraries/qt-5/hooks/qtbase-setup-hook.sh

```diff
@@ -1,3 +1,9 @@
+if (( "$targetOffset" <= 0 )); then
```

Comment:
Also follow strictDeps like cc-wrapper? https://github.com/NixOS/nixpkgs/blob/d7abb3a970302200f7a57075d64f837d855bdeba/pkgs/build-support/cc-wrapper/setup-hook.sh#L61

Reply:
No, I think it was a mistake to have `strictDeps` be opt-in.

Also, the `qtbase-setup-hook` is only allowed to run once per dependency outpath, but a dependency outpath can appear more than once with *different $hostOffset values*.

I think this check is too important to allow it to be muted by default.

Reply:
> Also, the `qtbase-setup-hook` is only allowed to run once per dependency outpath, but a dependency outpath can appear more than once with _different $hostOffset values_.

This means that the existing code is very sensitive to the order that dependencies appear in.  It's really bad.

Reply:
My bad, I thought the strictDeps part would make it stricter in some way, bash conditional....

Reply:
I found a better solution: compare the `stdenv` environment variable.  See latest push  e722e3e